flaky test: `chainerx_tests/unit_tests/routines_tests/test_linalg.py::test_SVD`

https://jenkins.preferred.jp/job/chainer/job/chainer_pr/2182/TEST=CHAINERX_chainerx-py3,label=mn1-p100/console

Occurred in #8233

>FAIL ../../../repo/tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py::test_SVD_param_5_{compute_uv=True, dtype='float32', full_matrices=False, shape=(3, 2)}[cuda:0]

```
19:00:04 E           chainer.testing.function_link.FunctionTestError: backward is not implemented correctly
19:00:04 E           
19:00:04 E           (caused by)
19:00:04 E           AssertionError: check_backward failed (eps=0.001 atol=1e-05 rtol=0.005)
19:00:04 E           inputs[0]:
19:00:04 E           array([[0.23005281, 0.06498358],
19:00:04 E                  [0.77840567, 0.14274246],
19:00:04 E                  [0.98044223, 0.33861399]], shape=(3, 2), dtype=float32, device='cuda:0')
19:00:04 E           grad_outputs[0]:
19:00:04 E           array([[0.6199522 , 0.36488649],
19:00:04 E                  [-0.21421909, 0.43409508],
19:00:04 E                  [-0.6566478 , -0.43067288]], shape=(3, 2), dtype=float32, device='cuda:0')
19:00:04 E           grad_outputs[1]:
19:00:04 E           array([0.57013792, -0.84386188], shape=(2,), dtype=float32, device='cuda:0')
19:00:04 E           grad_outputs[2]:
19:00:04 E           array([[0.51943040, 0.88441974],
19:00:04 E                  [0.84833151, 0.24966429]], shape=(2, 2), dtype=float32, device='cuda:0')
19:00:04 E           directions[0]:
19:00:04 E           array([[-0.06155252, 0.56848959],
19:00:04 E                  [0.13452480, 0.43904923],
19:00:04 E                  [0.02734724, 0.6792804 ]], shape=(3, 2), dtype=float64, device='cuda:0')
19:00:04 E           gradients (numeric):  array(-0.04954462, shape=(), dtype=float64, device='cuda:0')
19:00:04 E           gradients (backward): array(-0.04315267, shape=(), dtype=float64, device='cuda:0')
19:00:04 E           
19:00:04 E           x: numeric gradient, y: backward gradient
19:00:04 E           Not equal to tolerance rtol=0.005, atol=1e-05
19:00:04 E           
19:00:04 E           Mismatch: 100%
19:00:04 E           Max absolute difference: 0.00639195
19:00:04 E           Max relative difference: 0.1481241
19:00:04 E            x: array(-0.049545)
19:00:04 E            y: array(-0.043153)
19:00:04 E           
19:00:04 E           assert_allclose failed: 
19:00:04 E             shape: () ()
19:00:04 E             dtype: float64 float64
19:00:04 E             i: (0,)
19:00:04 E             x[i]: -0.04954462319472763
19:00:04 E             y[i]: -0.04315267238778159
19:00:04 E             relative error[i]: 0.1481241010870948
19:00:04 E             absolute error[i]: 0.006391950806946045
19:00:04 E             relative tolerance * |y[i]|: 0.00021576336193890794
19:00:04 E             absolute tolerance: 1e-05
19:00:04 E             total tolerance: 0.00022576336193890794
19:00:04 E           x: -0.04954462
19:00:04 E           y: -0.04315267
```

--- a/tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py
+++ b/tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py
@@ -112,10 +112,12 @@ class NumpyLinalgOpTest(op_utils.NumpyOpTest):
     dodge_nondifferentiable = True
 
     def setup(self):
+        super().setup()
         device = chainerx.get_default_device()
         if (device.backend.name == 'native'
                 and not chainerx.linalg._is_lapack_available()):
             pytest.skip('LAPACK is not linked to ChainerX')
+        self.check_forward_options.update({'rtol': 1e-4, 'atol': 1e-4})
         self.check_backward_options.update({'rtol': 5e-3})
         self.check_double_backward_options.update({'rtol': 5e-3})
 

